src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure figure1 = figureSupplier.getRandomFigure();
+        Figure figure2 = figureSupplier.getRandomFigure();
+        Figure figure3 = figureSupplier.getDefaultFigure();
+        Figure[] figures = new Figure[] {figure1, figure2, figure3};
```

Comment:
I do not see the second part of the array.
![image](https://user-images.githubusercontent.com/20326858/192279653-3835c4e4-cd23-4fd9-a400-2ef48f1fea93.png)
Also, better fill this array and draw figures in one time inside the loop.